app-text/cherrytree/Manifest

```diff
@@ -1,4 +1,4 @@
-DIST cherrytree-0.99.37.tar.gz 17983743 BLAKE2B 42e5464f4cba4364dbbdabe5daaf80517a62f3f4b2c1c3efa764e9b10dfae492bf335403d24a7a33d9b7c8cd55a0c55ad359ffca1f639fa7622e00a416defd60 SHA512 2413924d03c65dfad820bc2b1e345b7dd74b69d11a10ee0d333400936eb31368f53f2aee77a3dd708adb3d4b91578c0499766bca4c85537822e9c4b3d41c6800
 DIST cherrytree_0.99.47.tar.xz 2457904 BLAKE2B 1072de8ed62757a873a258dc9ab1e9e186128b520e7f02e0a2cd8e7e0173232af7db85e3293efd21b577e51b6f5e55cca0cf35631e34279e32dc07836335b742 SHA512 2611be4c24a188793aa6785d96b33ec91c0957b30616352e1cb2cd12f296e39c29666d37a03016be6c934898bf6deb98af74c2c345d9c70f138f62ac6b10f023
 DIST cherrytree_0.99.48.tar.xz 3474468 BLAKE2B b60cc2ffa4f5bd6e91e2a8b34811ad6977d46bdd02c7c1b2bfcdbf4df6107753a2390bc6a15daae8d5268af017ff52d217ab79ed5b4d5d107ef736257999b865 SHA512 eb4690bbc5865b9a0c7d0cb11341fe5ac17f11d417ad765b29784083a83f18c06a39b5b738838e3d57754de25adf1861ea4a6ad472e01a895e8fce0af0964926
```

Comment:
Drop all 3 versions too, if not required by specific packages, and ask for a stablereq of 0.99.51 when this PR will be merged. Having lots of versions of the same package is not always a good idea (except for some packages like mariadb or database manager in general)